rust/ql/lib/codeql/rust/elements/internal/LiteralExprImpl.qll

```diff
@@ -27,5 +27,7 @@ module Impl {
    */
   class LiteralExpr extends Generated::LiteralExpr {
     override string toString() { result = this.getTextValue() }
+
+    override string toAbbreviatedString() { result = this.getTextValue() }
```

Comment:
I now thought we might want to trim long literals here

Reply:
Yes